libs/iovm/source/IoSystem.c

```diff
@@ -22,7 +22,7 @@ Contains methods related to the IoVM.
 #if defined(__NetBSD__) || defined(__OpenBSD__)
 # include <sys/param.h>
 #endif
-#ifndef __CYGWIN__
+#ifndef __CYGWIN__ || defined(linux)
```

Comment:
This does not fix the Issue.
The line should only be used when both CYGWIN and linux are not defined.
```C
#if !defined(__CYGWIN__) && !defined(linux)
```

Reply:
It's more safe to do it this way:

```
#if defined(HAVE_SYS_SYSCTL_H) && !defined(__GLIBC__)
#include <sys/sysctl.h>
#endif
```

Also it allows it to include it if needed.